src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,102 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double GROW_INDEX = 1.5;
+    private static final String INDEX_OUT_OF_BOUNDS_EXEPTION = "This index "
+            + "out of bounds: ";
+    private static final String NO_SUCH_ELEMENT_EXEPTION = "Element not found in list ";
```

Comment:
No sure about this - we use those exeption only once so I dont see need to make them constants (Imho)

Reply:
But content of this messages not changing.
IMHO its resonable to make this strings CONSTANTs.